tests/console/udisks2.pm

```diff
@@ -0,0 +1,81 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Simple tests for udisks2 using udisksctl checking status,
+# block device information and loop-setup.
+# Maintainer: Michael Grifalconi <mgrifalconi@suse.com>
+
+use strict;
+use warnings;
+use base 'consoletest';
+use testapi;
+use utils;
+use version_utils qw(is_sle);
+
+sub run {
+    select_console 'root-console';
+
+    # Install udisks2 package. mkisofs and util-linux for support packages
+    if (is_sle('>=15')) {
+        zypper_call('in mkisofs udisks2 util-linux');
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check.